com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/pipeline/FontBuilder.java

```diff
@@ -23,41 +23,39 @@
 import com.dynamo.bob.font.Fontc;
 import com.dynamo.bob.fs.IResource;
 
-import com.dynamo.bob.pipeline.GlyphBankBuilder;
-
 import com.dynamo.render.proto.Font.FontDesc;
 import com.dynamo.render.proto.Font.FontMap;
 
 @BuilderParams(name = "Font", inExts = ".font", outExt = ".fontc")
-public class FontBuilder extends Builder<Void>  {
+public class FontBuilder extends Builder  {
```

Comment:
One of a very few places where I kept Builder, because I didn't figure out how to replace it for both GlyphBuilder and FontBuilder.